tla/consensus/ccfraft.tla

```diff
@@ -1671,4 +1671,20 @@ DebugCommittedEntriesTermsInv ==
             k <= commitIndex[i]
             => log[i][k].term >= log[j][k].term
 
+
+------------------------------------------------------------------------------
```

Comment:
@lemmy what is the benefit of moving this?

Reply:
done